arviz/utils.py

```diff
@@ -664,3 +665,36 @@ def credible_interval_warning(credible_interval, hdi_prob):
 
     hdi_prob = credible_interval
     return hdi_prob
+
+
+class HtmlTemplate:
+    """Contains html templates for InferenceData repr."""
```

Comment:
I think you'll need this to keep both black and numpydoc happy: https://github.com/arviz-devs/arviz/blob/master/arviz/rcparams.py#L28